backup-stores/azure/src/main/java/io/camunda/zeebe/backup/azure/FileSetManager.java

```diff
@@ -24,8 +24,8 @@
 import java.util.stream.Collectors;
 
 final class FileSetManager {
-  // The path format is constructed by partitionId/checkpointId/nodeId/nameOfFile
-  private static final String PATH_FORMAT = "%s/%s/%s/%s/";
+  // The path format is constructed by backups/partitionId/checkpointId/nodeId/nameOfFile
```

Comment:
:wrench: I'd use `contents` to stay consistent with the other stores. This will also help us eventually in factoring out shared code.